script.js

```diff
@@ -0,0 +1,81 @@
+function Book(title, author, pages,read) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.read = read;
+}
+
+const myLibrary = [
+    new Book("Heroes","Joe Abercrombie", 475, false), 
+    new Book("Metamorphosis", "Franz Kafka", 189, true),
+    new Book("Hobbit", "J.R.R. Tolkien", 259, true)];
+
+function listBook(book) {
+    let card = document.createElement("div");
+    card.setAttribute("data-index", `${myLibrary.indexOf(book)}`)
+    card.classList.add("card");
+    
+    card.innerHTML = `
```

Comment:
Suggestion: Move the HTML partial to an outside funtion.
Something like:
```js
card.innerHTML = buildBookCardHtml(book, myLibrary);
```

Reply:
Yeah, I agree, a block of HTML looks messy inside other code